src/Bridge/Symfony/Resources/config/flash.php

```diff
@@ -24,23 +24,21 @@
 
         ->set('sonata.twig.extension.flashmessage.class', FlashMessageExtension::class);
 
-    // Use "service" function for creating references to services when dropping support for Symfony 4.4
-    // Use "param" function for creating references to parameters when dropping support for Symfony 5.1
     $containerConfigurator->services()
 
         ->set('sonata.twig.flashmessage.manager', '%sonata.twig.flashmessage.manager.class%')
             ->public()
             ->tag('sonata.status.renderer')
             ->args([
-                new ReferenceConfigurator('request_stack'),
+                service('request_stack'),
                 [],
```

Comment:
if this is an abstract argument it can be replaced too, abstract_arg('explanation of argument')

Reply:
Done